src/azure-cli-core/azure/cli/core/azlogging.py

```diff
@@ -77,10 +77,15 @@ def _delete_old_logs(log_dir):
 
             cmd_logger = logging.getLogger(AzCliLogging._COMMAND_METADATA_LOGGER)
 
-            self._init_command_logfile_handlers(cmd_logger, args)  # pylint: disable=protected-access
-            get_logger(__name__).debug("metadata file logging enabled - writing logs to '%s'.", self.command_log_dir)
+            # overwrite CLILogging._is_file_log_enabled() from knack
+            self.file_log_enabled = cli_ctx.config.getboolean('logging', 'enable_log_file', fallback=True)
```

Comment:
The default for logging is no in az configure.

Reply:
However, changing it to no could be a breaking change since the bug made the actual default to be logging enabled.

Reply:
I overwrite the implementation from knack to enable logging by default like currenty CLI does, so it's transparency to user. Not a breaking it is.